cmd/lncli/invoicesrpc_active.go

```diff
@@ -38,8 +38,11 @@ var settleInvoiceCommand = cli.Command{
 	Usage:    "Reveal a preimage and use it to settle the corresponding invoice.",
 	Description: `
 	Todo.`,
-	ArgsUsage: "preimage",
```

Comment:
shouldn't this be updated rather than removed?

Reply:
reverted to keep cmd line usage as much the same as possible